appveyor.yml

```diff
@@ -2,7 +2,7 @@ version: 1.0.{build}
 image:
 - Visual Studio 2015
 environment:
-  LIB_VERSION: 1.1.7 #Hardcode the LIB_VERSION : should be retrieved by building libcore node module and run tests/lib_version.js
+  LIB_VERSION: 2.0.0 #Hardcode the LIB_VERSION : should be retrieved by building libcore node module and run tests/lib_version.js
```

Comment:
For this I would like to do same as circleCI, build the node module and retrieve the version, this will allow us to simply test NodeJS binding, I tried before but I was having trouble with node on Appveyor